### DISCUSSION TITLE
Conditional independence symbol when rendering to pdf

### DISCUSSION THREAD

**Discussion by jfieberg** [quarto-dev/quarto-cli#11447] [Q&A] [answered]
I am trying to compile a quarto document to both HTML and pdf that includes a conditional independence symbol and a "not" version (see below).  I've tried several options, but can only seem to get things to work when compiling to HTML.  Here is the quarto file:

````qmd
---
title: "Reproducible Quarto Document"
format: pdf
---


 
...two types of associations between $X$ and $Y$, with and without conditioning on a third variable, $Z$. In the top panels, $X$ and $Y$ are marginally dependent (i.e., $X  \not\!\perp\!\!\!\perp Y$), but become independent when we condition on $Z$ (i.e., $X \perp\!\!\!\perp Y | Z$).  
````

Here is the output in the terminal window:

````
==> quarto preview test2.qmd --to pdf --no-watch-inputs --no-browse

pandoc 
  to: latex
  output-file: test2.tex
  standalone: true
  pdf-engine: xelatex
  variables:
    graphics: true
    tables: true
  default-image-extension: pdf
  
metadata
  documentclass: scrartcl
  classoption:
    - DIV=11
    - numbers=noendperiod
  papersize: letter
  header-includes:
    - \KOMAoption{captions}{tableheading}
  block-headings: true
  

Rendering PDF
running xelatex - 1
  This is XeTeX, Version 3.141592653-2.6-0.999996 (TeX Live 2024) (preloaded format=xelatex)
   restricted \write18 enabled.
  entering extended mode
  
updating tlmgr

updating existing packages
ERROR: 
compilation failed- error
Missing { inserted.
<to be read again> 
                   \mskip 
l.164 \(X  \not\!
                 \perp\!\!\!\perp Y\)), but become independent when we 

see test2.log for more information.
````

I am using quarto version 1.6.33 and Microsoft Windows [Version 10.0.26100.2033]. Here is the output from quarto check:

````bash
C:\Users\jfieberg\Documents\Teaching\Sandbox>quarto check
Quarto 1.6.33
[>] Checking environment information...
      Quarto cache location: C:\Users\jfieberg\AppData\Local\quarto
[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.4.0: OK
      Dart Sass version 1.70.0: OK
      Deno version 1.46.3: OK
      Typst version 0.11.0: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.6.33
      Path: C:\Users\jfieberg\AppData\Local\Programs\Quarto\bin
      CodePage: 1252

[>] Checking tools....................OK
      TinyTeX: (external install)
      Chromium: (not installed)

[>] Checking LaTeX....................OK
      Using: TinyTex
      Path: C:\Users\jfieberg\AppData\Roaming\TinyTeX\bin\windows\
      Version: 2024

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....(None)

      Unable to locate an installed version of Python 3.
      Install Python 3 from https://www.python.org/downloads/

[>] Checking R installation...........OK
      Version: 4.4.1
      Path: C:/Users/jfieberg/AppData/Local/Programs/R/R-4.4.1
      LibPaths:
        - C:/Users/jfieberg/AppData/Local/Programs/R/R-4.4.1/library
      knitr: 1.48
      rmarkdown: 2.27

[>] Checking Knitr engine render......OK
````

**Comment by mcanouil** [comment]
Could you properly format your post using code blocks for code and terminal outputs? Thanks.
If your code contains code blocks, you need to enclose it using more backticks, *i.e.*, usually four ` ```` `.
See <https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us>.

**Reply by jfieberg** [reply]
Thanks.  I don't have any code blocks in the file, only text/latex code.  I'm not sure I formatted the output correctly. Happy to try again. Thanks!

**Reply by mcanouil** [reply]
Your entire post is a code block right now.
Separate your comments, etc. from the code to reproduce your issues.
Don't mixe everything together which makes it hard to know what is what.


---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Reply by jfieberg** [reply]
Thanks for your patience! I *think* I got it this time (sorry for the confusion on my part!).

**Comment by cderv** [comment, marked as answer]
I think this is LaTeX problem directly. Quarto defaults to xelatex and this is what causes the issue. 

If you set `pdf-engine: pdflatex` it works

````markdown
---
title: "Reproducible Quarto Document"
format: 
  pdf:
    pdf-engine: pdflatex
---


 
...two types of associations between $X$ and $Y$, with and without conditioning on a third variable, $Z$. In the top panels, $X$ and $Y$ are marginally dependent (i.e., $X  \not\!\perp\!\!\!\perp Y$), but become independent when we condition on $Z$ (i.e., $X \perp\!\!\!\perp Y | Z$).  
````

So I would look into why this specific latex equation syntax for xelatex or lualatex engine. 

Using `pandoc -t pdf -f markdown -o test.pdf --pdf-engine=xelatex test.qmd` gives me the same error so I don't think this is related to Quarto directly unfortunately.

**Reply by jfieberg** [reply]
Thanks!  I'm happy to switch to pdflatex as that seems to solve the problem!
